src/definitions/modules/slider.js

```diff
@@ -138,6 +138,9 @@ $.fn.slider = function(parameters) {
           slider: function() {
             sliderObserver.observe($module[0], {
```

Comment:
Do we still need to have an observer? I can't think of anything in the class or style that would change where we would need to re-sync everything as all the positioning is percentage based.